examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -80,6 +85,12 @@ class _ButtonsDemoState extends State<ButtonsDemo> {
         widget: buildActionButton(),
         exampleCodeTag: _actionCode,
       ),
+      new ComponentDemoTabData(
+        tabName: 'iOS',
+        description: _iosText,
```

Comment:
IMHO, we should use "cupertino" consistently in the code when referring to these widgets.